### ISSUE TITLE
[CAIA A11y Research]: Pension 527EZ Form Updates - Benefits & Claims (2024 - Month TBD)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#66842]
`Pending Invites - Possible Feb. 2024 Start`

### Who's on your team?

```bash
Team name: Benefits and Claims | Lifestage Benefits Crew | Non-Disability Experience
OCTO-DE product owner: Emily Theis
Product manager: Laura Steele
Designer: Fiorella Glavez
Accessibility specialist: none
Content writer: none
Slack channel: benefits-non-disability
...
```

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [ ] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [ ] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [ ] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

2024

### Supporting artifacts (if any)

- Product outline: [Pension Online Form Reactivation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/527EZ-reactivation-initiative-brief.md)
- User flows: [Mural of user flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1695069026901/50b83f710f1cd2f1f7f3bb63553c349ef90d402a?sender=u4e1fb0748c58370c03f33409) | [Mural of form flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1683826730324/61b509f935adaa2acf2f7769a5ba911726a7b237?sender=u0e8ac1d6d7681bb7e1b80558)
- Prototype or mockups: [Pension redesign](https://www.sketch.com/s/75b363db-6be8-4b7e-8655-8f12f1ea5554) | [Pension entry points](https://sketch.com/s/42d1dd07-4ed9-4330-b732-c2a8f183e6f3)
- Content source of truth: [21P-527EZ paper form](https://www.vba.va.gov/pubs/forms/VBA-21P-527EZ-ARE.pdf)
- Research plan: N/A
- Any epics/issues that may be helpful: https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349
- Link to testing accounts (please do not paste them in this ticket): [527EZ test accounts](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/form-deactivation/527EZ-staging-test-accounts-accessible.md)

### Next steps

- [ ] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [ ] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [ ] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [ ] CAIA Specialists review your conversation guide and research plan
- [ ] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [ ] CAIA Specialists create a research notes template
- [ ] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by sara-amanda** [comment]
Happy 2024, @steele-lm ! Just checking in to follow up on your upcoming research fo the Pension 527EZ for Updates for Benefits and Claims, and to see how CAIA might assist you in your AT sessions. My notes say possible Feb. 2024 start, so just wanting to confirm or update, based on your feedback. Thanks so much!

@coforma-terry @coforma-jamie

**Comment by steele-lm** [comment]
@sara-amanda My team handed off the pension form updates and future work to the new Pension Team, so they will be carrying any future research forward! @vbellissimo is the product manager for that team, and I believe they are still on track to launch the initial round of critical changes by the end of January.

**Comment by sara-amanda** [comment]
Happy Friday, @vbellissimo !
- Do you know if this recent intake ticket for Direct Deposit: Ticket #77464 is a part of this future `TBD` research? 
- Would you be open to syncing with @coforma-terry and myself from CAIA on outstanding Pension-related tickets with CAIA? 
cc: @EvanAtCoforma

**Comment by coforma-jamie** [comment]
Since it's been more than 3 months, we're going to close this ticket. The CAIA accessibility team is moving to a new contract soon - if you have research needs, please open a new ticket at that point.
